File: docs/mexc/mexc-api-sdk-main/mexc-api-sdk-main/examples/python/cancel_order_with_external_id.py
from mexc_client import MexcClient

client = MexcClient(api_key='YOUR_API_KEY', is_testnet=True)

result = client.cancel_order_with_external_id({
    'symbol': "BTC_USDT",
    'externalOid': "mexc-a-001"
})

print(result)

File: docs/mexc/mexc-api-sdk-main/mexc-api-sdk-main/examples/nodejs/cancelOrderWithExternalId.js
import { MexcClient } from './MexcClient.js';

const client = new MexcClient({ apiKey: 'YOUR_API_KEY', isTestnet: true });

const result = await client.cancelOrderWithExternalId({
  symbol: "BTC_USDT",
  externalOid: "mexc-a-001"
});

console.log(result);